test/e2e/apps/network_partition.go

```diff
@@ -40,6 +40,7 @@ import (
 	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
 	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	e2ess "k8s.io/kubernetes/test/e2e/framework/statefulset"
```

Comment:
Thoughts on `e2esset`?

Reply:
It is fine to make these import consistent with a nice @dims work of https://github.com/kubernetes/kubernetes/pull/78836 later.

Reply:
Yeah anything but e2ess would be better. 
e2estateful 
e2esset <- +1